.travis.yml

```diff
@@ -1,22 +1,15 @@
-language: erlang
+language: elixir
 
 otp_release:
   - 17.1
 
-env:
-  - ELIXIR_VERSION=1.0.0-rc1
-  - ELIXIR_VERSION=1.0.0-rc2
-  - ELIXIR_VERSION=1.0.0
-
-install:
-  - wget https://github.com/elixir-lang/elixir/releases/download/v$ELIXIR_VERSION/Precompiled.zip
-  - unzip -d elixir Precompiled.zip
+elixir:
+  - 1.0.0-rc1
+  - 1.0.0-rc2
```

Comment:
No longer need these old release candidates.  I guess just change to the current stable versions?